client/app/scripts/superdesk-publish/publish.js

```diff
@@ -184,7 +184,7 @@
         function fetchPublishQueue () {
             var criteria = criteria || {};
             criteria.max_results = 200;
-            criteria.sort = '[(\'_created\',-1)]';
+            criteria.sort = '[(\'published_seq_num\',-1)]';
```

Comment:
What about replacing the internal single quotes that are now escaped? Would that work or would some other component complain that it cannot parse the sort criterion due to the double quotes? Just an idea ...